build/cqs.tsv

```diff
@@ -244,8 +244,8 @@ org.apache.hadoop:hadoop-distcp	2.6.4	provided
 org.apache.hadoop:hadoop-mapreduce-client-core	2.6.4	provided
 org.apache.hadoop:hadoop-yarn-api	2.6.4	provided
 org.apache.hadoop:hadoop-yarn-common	2.6.4	provided
-org.apache.kafka:kafka_2.11	0.9.0.1	provided
-org.apache.kafka:kafka-clients	0.9.0.1	provided
+org.apache.kafka:kafka_2.11	1.0.1	provided
+org.apache.kafka:kafka-clients	1.0.1	provided
```

Comment:
our current CQ is still valid: https://dev.eclipse.org/ipzilla/show_bug.cgi?id=9082